.changelog/4445.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: replacing playbookId with playbookName in tasks.
```

Comment:
```suggestion
- description: replacing playbookId with playbookName in tasks.
```
Should be in the template of Fixed an issue where...

Reply:
@maimorag See https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs for all available templates.